examples/Demo/Shared/Components/SiteSettingsPanel.razor

```diff
@@ -7,11 +7,10 @@
             <FluentSelect Id="color"
                           Items=@(Enum.GetValues<OfficeColor>())
                           OptionValue="@(c => c.ToAttributeValue())"
-                          OptionSelected="@(c => c == _selectedColorOption)"
+                          OptionSelected="@(c => c == ThemeService.SelectedAccentColor)"
                           TOption="OfficeColor"
                           Position="SelectPosition.Below"
-                          @bind-SelectedOption="@_selectedColorOption"
-                          @onchange="@(x => HandleColorChange(x))"
+                          @onchange="@(x => ThemeService.SetAccentColorAsync(x.Value!.ToString()!.GetEnumByDescription<OfficeColor>()))"
```

Comment:
Could you set the code in the .cs file. So, it's better to create a `HandleThemeChange` method with this code.

Reply:
I'll make sure to follow that in the future PRs and update the current one as well